.github/workflows/go-release.yaml

```diff
@@ -199,3 +164,41 @@ jobs:
               console.error(error);
               core.setFailed(error);
             }
+
+  release-binary:
+    name: Release Binary
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+          path: go/src/open-cluster-management.io/multicluster-controlplane
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - name: generate changelog
+        run: |
+          echo "# Controlplane ${{ needs.env.outputs.RELEASE_VERSION }}" > /home/runner/work/changelog.txt
+          echo "- See the [CHANGELOG](https://github.com/open-cluster-management-io/multicluster-controlplane/CHANGELOG/CHANGELOG-${{ needs.env.outputs.MAJOR_RELEASE_VERSION }}.md) for more details." >> /home/runner/work/changelog.txt
```

Comment:
the path is not correct. it should like `https://github.com/open-cluster-management-io/multicluster-controlplane/blob/main/CHANGELOG/CHANGELOG-v0.2.md`